### PR TITLE
mesh_announce: add missing wireguard interfaces

### PR DESCRIPTION
**Pull request by AiyionPrime**
This adds the wireguard-setup related interfaces to mesh-announce.

### CODE REVIEW

**Comment by AiyionPrime** [comment]
Though this must be a part of the solution, mesh-announce does not encounter the multicats message on vx-14, though it's sent on a regualr basis and received, as wireshark confirms.

**Comment by AiyionPrime** [comment]
https://github.com/freifunkh/ansible/blob/b418009ce1f4468a8bf5e7ec1ab0879402f4e294/roles/ffh.mesh_announce/templates/ferm.conf.j2

**Comment by AiyionPrime** [comment]
Tested on sn09

**Review by 1977er — APPROVED** [review]
lgtm

**Comment by AiyionPrime** [comment]
Thanks @1977er and @lemoer .

**Comment by AiyionPrime** [comment]
wait for it... wait for it...

**Comment by AiyionPrime** [comment]
The added line
`saddr fe80::/64 sport 1001 dport 32768:61000 mod comment comment "mesh_respondd_reply" ACCEPT;`
translates to
`-A MESH_INPUT -s fe80::/64 -p udp -m udp --sport 1001 --dport 32768:61000 -m comment --comment mesh_respondd_reply -j ACCEPT`

Which corresponds to the routers firewall rule:
> `-A zone_mesh_input -s fe80::/64 -p udp -m udp --sport 1001 --dport 32768:61000 -m comment --comment "!fw3: mesh_respondd_reply" -j ACCEPT`
_Originally posted by @AiyionPrime in https://github.com/freifunkh/ansible/issues/195#issuecomment-801873126_

**Comment by AiyionPrime** [comment]
Ah yeah, still tested on sn09.

File: roles/ffh.mesh_announce/templates/ferm.conf.j2 [hunk 1]
```diff
@@ -1,7 +1,10 @@
 domain (ip ip6) {
   table filter {
     chain MESH_INPUT {
-      proto udp dport (1001) ACCEPT;
+      proto udp {
+        dport (1001) ACCEPT;
+        saddr fe80::/64 sport 1001 dport 32768:61000 mod comment comment "mesh_respondd_reply" ACCEPT;
```
Comment (lemoer) [inline roles/ffh.mesh_announce/templates/ferm.conf.j2:6]: Can we remove `dport` here?

Comment (lemoer) [reply roles/ffh.mesh_announce/templates/ferm.conf.j2:6]: Why is the line necessary at all? This should be handled by conntrack anyways, shouldn't it?

Comment (AiyionPrime) [reply roles/ffh.mesh_announce/templates/ferm.conf.j2:6]: > Can we remove dport here?

Yes, we could. But this is more specific.

> This should be handled by conntrack anyways, shouldn't it?

I can assure it is not optional. I'm not sure, whether conntrack is clever enough for this; effectively it's an 'unrelated' unicast answer to a multicast query.

Comment (AiyionPrime) [reply roles/ffh.mesh_announce/templates/ferm.conf.j2:6]: Have you seen the comment above?
https://github.com/freifunkh/ansible/pull/197#issuecomment-801935031

**Pull request merged**